### PR TITLE
Motorist profile feature

### PR DESCRIPTION
**Pull request by pandwoter**
Added possibility to fill basic information about motorist and the service owner

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -34,16 +37,19 @@ gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 # CUSTOM GEMS #
 # ########### #
 gem 'bootstrap', '~> 4.3.1'
-gem 'devise'
-gem 'simple_form'
-gem 'country_select'
+
+##ADMIN UI
```
Comment (dmokreckiy) [inline Gemfile]: :+1:

File: Gemfile [hunk 2]
```diff
@@ -34,16 +37,19 @@ gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
...
 gem 'rails_admin'
+##FORM HELPERS
```
Comment (dmokreckiy) [inline Gemfile]: :+1:

File: Gemfile [hunk 3]
```diff
@@ -34,16 +37,19 @@ gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
...
+gem 'country_select'
+gem 'simple_form'
+##AUTH AND REGISTATION GEMS
```
Comment (dmokreckiy) [inline Gemfile]: :+1:
REGIST**R**ATION

File: Gemfile [hunk 4]
```diff
@@ -34,16 +37,19 @@ gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
...
 gem 'dotenv-rails'
-gem 'byebug'
-
-
-
-
+##IMAGE UPLOADING AND PROCESSING
+gem 'carrierwave'
+gem 'mini_magick'
+################################
```
Comment (dmokreckiy) [inline Gemfile]: New line missing

File: app/helpers/application_helper.rb [hunk 5]
```diff
@@ -1,2 +1,15 @@
 module ApplicationHelper
+  def current_model
```
Comment (dmokreckiy) [inline app/helpers/application_helper.rb]: Bad naming

File: app/helpers/application_helper.rb [hunk 6]
```diff
@@ -1,2 +1,15 @@
...
+    current_user || current_service_owner
+  end
+
+  def destroy_signed_in_session
+    case current_model.class.name
```
Comment (dmokreckiy) [inline app/helpers/application_helper.rb]:
```
case current_model.class
when User then ...
when Motorist then ...
...
else
 ...
end
```

File: app/models/abstract_user.rb [hunk 7]
```diff
@@ -1,7 +1,17 @@
...
-  # Include default devise modules. Others available are:
-  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable, :confirmable
 
+  mount_uploader :profile_picture, ImageUploader
+
+  def db_table
+    case self.type
```
Comment (dmokreckiy) [inline app/models/abstract_user.rb]: https://devdocs.io/rails~5.2/activemodel/naming use `model_name` on `self.class`

File: app/helpers/application_helper.rb (hunk 6 above)
Comment (pandwoter) [reply app/helpers/application_helper.rb]: does not work btw

File: Gemfile (hunk 4 above)
Comment (pandwoter) [reply Gemfile]: fixed

File: Gemfile (hunk 3 above)
Comment (pandwoter) [reply Gemfile]: fixed

File: app/models/abstract_user.rb (hunk 7 above)
Comment (pandwoter) [reply app/models/abstract_user.rb]: fixed

**Pull request merged**